src/Dynatrace.OpenTelemetry.Exporter.Metrics.Tests/DynatraceMetricsExporterTests.cs

```diff
@@ -22,17 +22,19 @@
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
 using Moq;
 using Moq.Protected;
 using OpenTelemetry.Metrics.Export;
 using Xunit;
+using static System.Threading.CancellationToken;
 
 namespace Dynatrace.OpenTelemetry.Exporter.Metrics.Tests
 {
 	public class DynatraceMetricsExporterTests
 	{
 		[Fact]
-		public async void TestDefaultOptions()
+		public async Task ExportAsync_WithDefaultOptions_ShouldSendRequestToOneAgent()
```

Comment:
Because the tests are async, now the `.editorconfig` is producing warnings that the func name must end with `Async`. 

We can solve it by adding a new file inside the test project: `GlobalSuppressions.cs` with this content:

```csharp
// <copyright company="Dynatrace LLC">
// Copyright 2020 Dynatrace LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
// </copyright>

// This file is used by Code Analysis to maintain SuppressMessage
// attributes that are applied to this project.
// Project-level suppressions either have no target or are given
// a specific target and scoped to a namespace, type, member, etc.

using System.Diagnostics.CodeAnalysis;

[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "Test project")]

```

Reply:
Good point! Did not see it in vscode :+1: